src/input-schemas.jl

```diff
@@ -29,6 +29,13 @@ const schemas = (
             :use_binary_storage_method => "VARCHAR",         # Whether to use an extra binary variable for the storage assets to avoid charging and discharging simultaneously (missing;binary;relaxed_binary)
             :max_energy_timeframe_partition => "DOUBLE",     # MWh (Missing -> no limit)
             :min_energy_timeframe_partition => "DOUBLE",     # MWh (Missing -> no limit)
+            :unit_commitment => "BOOLEAN",                   # Whether asset has unit commitment constraints
```

Comment:
Maybe it's better to change this to `has_unit_commitment`?

Reply:
Though, we have more of these variables, and we don't follow a convention, so maybe let's create an issue to deal with them later.

Reply:
I agree, We need to establish a clear convention for the names of the option parameters. I like the `has_XXX` true or false. This is also related to #622. I will add a comment about this there.